test/dfusionMultiSafes.js

```diff
@@ -364,76 +257,102 @@ contract("GnosisSafe", function(accounts) {
           exchange,
           stableToken,
           targetToken,
-          depositAmountStableToken.div(new BN(1)),
-          depositAmountTargetToken.div(new BN(3))
+          bracketsWithStableTokenDeposit == 0 ? 0 : depositAmountStableToken.div(new BN(bracketsWithStableTokenDeposit)),
+          bracketsWithTargetTokenDeposit == 0 ? 0 : depositAmountTargetToken.div(new BN(bracketsWithTargetTokenDeposit))
         )
       }
+    }
+    it("transfers tokens from fund account through trader accounts and into exchange via automatic deposit logic, p > 1", async () => {
+      const testEntries = [
+        {
+          tradeInfo: {
+            fleetSize: 4,
+            lowestLimit: 100,
+            highestLimit: 121,
+            currentPrice: 110,
+            amountStableToken: "0.000000000000001",
+            amountTargetToken: "0.000000000000002",
+            stableTokenInfo: { decimals: 18, symbol: "DAI" },
+            targetTokenInfo: { decimals: 18, symbol: "WETH" },
+          },
+          expectedDistribution: {
+            bracketsWithStableTokenDeposit: 2,
+            bracketsWithTargetTokenDeposit: 2,
+          },
+        },
+      ]
+      await Promise.all(
```

Comment:
This seems to be a little bit of overkill. A simple await for the `testAutmaticDeposits` should have been sufficient?